views/networking/load_balancer/overview.erb

```diff
@@ -15,13 +15,13 @@
 ]
 
 unless @lb.health_check_protocol == "tcp"
-  data << ["HTTP Health Check Endpoint", @lb.health_check_endpoint]
-  data << ["SSL Certificate Status", @lb.active_cert&.cert ? "Available" : "Creating"] if @lb.health_check_protocol == "https"
+  data << ["#{@lb.health_check_protocol.upcase} Health Check Endpoint", @lb.health_check_endpoint]
+  data << ["SSL Certificate Status", @lb.active_cert&.cert ? "Available" : "Creating"] if @lb.cert_enabled
 end
```

Comment:
Since you want to allow SSL certs even for tcp load balancers:

```suggestion
end
data << ["SSL Certificate Status", @lb.active_cert&.cert ? "Available" : "Creating"] if @lb.cert_enabled
```